aries-site/src/pages/templates/forms.mdx

```diff
@@ -216,11 +200,60 @@ For standard forms, use an asterisk(\*) to indicate required field/controls.
   <SingleFormFieldExample />
 </Example>
 
-## Customize
+### Customize
 
 <Example
   code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/forms/CustomizeExample.js"
   figma="https://www.figma.com/file/3fkwBelW5UsCbfwdDCJkT8/HPE-Form-Templates?node-id=54%3A608"
 >
   <CustomizeExample />
 </Example>
+
+### Character counter
+
+**When to use**  
+In cases where the user is given an opportunity to express themselves or would likely exceed a field's character limit. Examples include some description fields, feedback, feature requests, reporting issues, etc.
+
+**Guidance**
+
+- Place the character counter at the bottom right of the field.
+- [See guidance on character limits for fields](https://design-system.hpe.design/components/tag#anatomy). Note that your backend may have character limitations. When you can choose the character limits, however, set a generous number to avoid frustrating the user. A suggestion to ensure your limit is user-friendly, check the analytics to see how long it is taking your users to complete this task. Longer than usual time could indicate the character limits are too low. Increase accordingly.
+- Use a numerator/denominator format. Ex. 0/128.
+- Show the progression in the numerator as the user is typing. Ex. 52/128 if the user has typed 52 characters, 53/128 when they type an extra character, 54/128, 55/128, etc.
+- Use default text styling.
+- Make the field roughly the same size as the character limit.
+- Red highlight the field when the user meets the character limitation.
+
+**Accessibility**
+
+- Announce the max character allowed before the user begins typing.
+- Announce when the user has typed 50% of the max allowed.
+
+### Accessibility
+
+All form input elements must be contained within a FormField.
```

Comment:
Why?

We have an opportunity to be educating here, and should be. This and the following "musts" reads as directives, but is lacking explanation for the importance of why this should be done.

Opportunity to explain why doing, what needs to be done, and what it accomplishes.

Reply:
Something like this might work:

"Forms need to be presented in an accessible manner for all users. Accordingly, all form inputs need to be accompanied by a label which identifies itself to screen readers. Placing all inputs within a Grommet FormField and applying the following play a big role in ensuring the inputs are accessible:"